src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -13112,5 +13112,88 @@ .maxstack  1
+                        }
+
+                        class N
+                        {
+                        }
+                        """,
+                    edits: new[] {
+                        Edit(SemanticEditKind.Delete, symbolProvider: c => c.GetMember("C.M1"), containingTypeProvider: c => c.GetMember("C")),
+                        Edit(SemanticEditKind.Delete, symbolProvider: c => c.GetMember("C.M2"), containingTypeProvider: c => c.GetMember("C")),
+                        //Edit(SemanticEditKind.Delete, symbolProvider: c => c.GetMember("C.M3"), containingTypeProvider: c => c.GetMember("C")),
+                        //Edit(SemanticEditKind.Delete, symbolProvider: c => c.GetMember("C.M4"), containingTypeProvider: c => c.GetMember("C")),
+                    },
+                    validator: g =>
+                    {
+                        g.VerifyTypeDefNames();
+                        g.VerifyMethodDefNames("M1", "M2" /*, "M3", "M4"*/);
+                        g.VerifyEncLogDefinitions(new[]
+                        {
+                            Row(1, TableIndex.MethodDef, EditAndContinueOperation.Default),
+                            Row(2, TableIndex.MethodDef, EditAndContinueOperation.Default),
+                            //Row(3, TableIndex.MethodDef, EditAndContinueOperation.Default),
+                            //Row(4, TableIndex.MethodDef, EditAndContinueOperation.Default),
+                            Row(1, TableIndex.Param, EditAndContinueOperation.Default),
+                            //Row(2, TableIndex.Param, EditAndContinueOperation.Default),
+                            //Row(3, TableIndex.Param, EditAndContinueOperation.Default)
+                        });
+                        g.VerifyEncMapDefinitions(new[]
+                        {
+                            Handle(1, TableIndex.MethodDef),
+                            Handle(2, TableIndex.MethodDef),
+                            //Handle(3, TableIndex.MethodDef),
+                            //Handle(4, TableIndex.MethodDef),
+                            Handle(1, TableIndex.Param),
+                            //Handle(2, TableIndex.Param),
+                            //Handle(3, TableIndex.Param),
+                        });
+
+                        // TODO: This should be throwing MissingMethodException
+                        var expectedIL = """
+                            {
+                              // Code size        2 (0x2)
+                              .maxstack  8
+                              IL_0000:  ldnull
+                              IL_0001:  throw
+                            }
+                            """;
+
+                        // Can't verify the IL of individual methods because that requires IMethodSymbolInternal implementations
+                        // TODO: This should probably output more than just one method worth of IL, right?
+                        g.VerifyIL(expectedIL);
```

Comment:
@tmat Am I right in thinking this is a red flag, that I'm "emitting" two methods, but only one block of IL is in the delta? Or is there some smart de-duping going on?

I wouldn't be surprised if its the former, thats for sure, I just need to work out which property of `IMethodDefinition` or `IMethodBody` is causing it :)